docs/cdoc/qk-target-entry.rst

```diff
@@ -0,0 +1,37 @@
+
+=============
+QkTargetEntry
+=============
+
+.. code-block:: C
+
+    typedef struct QkTargetEntry QkTargetEntry
+
+A mapping of qargs and instruction properties representing gate map of the 
+Target. This feature is used due to not having valid native mappings available from
+C.
+
+Here's an example of how this structure works:
+
+.. code-block:: C
+
+    #include <qiskit.h>
+    #include <math.h>
+
+    // Create a Target Entry for a CX Gate
+    QkTargetEntry *entry = qk_target_entry_new(QkGate_CX);
+
+    // Add mapping between (0,1) and InstructionProperties(10e-9, NAN)
```

Comment:
It would be nice to expand a bit here, but this can be left as follow up (like the other comment on docs)